src/services/UserService.js

```diff
@@ -7,15 +7,16 @@ export default class UserService {
         }
         return this.instance
     }
+
     findUserById = userId =>
-        fetch(MIDDLE_TIER_URL + '${userId}')
+        fetch(MIDDLE_TIER_URL + '/api/users/' + `${userId}`)
             .then(response => response.json())
-    findAllUsers = () =>
-        fetch(MIDDLE_TIER_URL)
+    findAllUser = () =>
```

Comment:
any reason you removed the s anywhere? might be missing something but this just seems like a random change

Reply:
Because the function is called "findAllUser" in the backend, not "findAllUsers"